CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## master / unreleased
+* [CHANGE] Alertmanager: Validating new fields on the Webhook AM config. #5276
```

Comment:
Since this update includes multiple changes from AM and the validation change, shall we mention more update/changes we get?

Reply:
sure will include more changes in the change log.